moped-toolbox/backfill_project_lead/run.py

```diff
@@ -0,0 +1,171 @@
+"""Updates the `project_lead_id` column of moped_project records based on the the user who created the project.
```

Comment:
no harm pushing this to prod now, just wondering if you meant to include this along for the ride

Reply:
ah crap i branched from main